website/developer-docs/docs/templates/index.md

```diff
@@ -16,7 +16,7 @@ The most common types are:
 
 ### Add a new integration
 
-To add documentation for a new [integration](../../../integrations/) (with support level Community or Vendor), please use the integration template [`service.md`](https://github.com/goauthentik/authentik/blob/main/website/integrations/_template/service.md) file from our GitHub repo. You can download the template file using the following command:
+To add documentation for a new integration (with support level Community or Vendor), please use the integration template [`service.md`](https://github.com/goauthentik/authentik/blob/main/website/integrations/_template/service.md) file from our GitHub repo. You can download the template file using the following command:
```

Comment:
Nitpick: the word "file" in "... file from out GitHub repo" is redundant.  A template is a file.  

Also, I'm not entirely sure I want a user who needs help using wget using our product.  Most people will just click the link and do "Save as...", I suspect.

Reply:
I removed "file" but left the `wget` command... I want our users to learn and also make sure we are friendly to new users.